lib/workos/errors.rb

```diff
@@ -18,16 +21,27 @@ class WorkOSError < StandardError
         http_status: T.nilable(Integer),
         request_id: T.nilable(String),
         code: T.nilable(String),
+        errors: T.nilable(T::Array[T::Hash[T.untyped, T.untyped]]),
       ).void
     end
-    def initialize(message: nil, error: nil, error_description: nil, http_status: nil, request_id: nil, code: nil)
+    def initialize(
+      message: nil,
+      error: nil,
+      error_description: nil,
+      http_status: nil,
+      request_id: nil,
+      code: nil,
+      errors: nil
```

Comment:
Does it make sense to have both `error` and `errors`?

Reply:
This is a base class for all our error types. `error` and `error_description` are from [OAuth related errors](https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_flow_errors.htm&type=5), while `code`, `message` and `errors` are custom props that we've defined for WorkOS errors.